apps/01PigLatin.js

```diff
@@ -9,15 +9,50 @@ var rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  word = word.toLowerCase();
+  var firstLetter = word[0];
+  console.log(firstLetter);
+  //
 
-  // Your code here
+  var vowelIndex = -1;
+  if ((word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('a');
+  }
+  if ((word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('e');
+  }
+  if ((word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('i');
+  }
+  if ((word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('o');
+  }
+  if ((word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('u');
+  }
+  if ((word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('y');
+  }
```

Comment:
Great work! try taking a look at your classmates' pull requests for some ideas on how you might write this app differently (James, Leslie and Tony had some good stuff)